packages/foam-vscode/src/features/janitor.ts

```diff
@@ -149,14 +156,14 @@ async function runJanitor(foam: Foam) {
         }
 
         if (heading) {
-          console.log("editor.write heading " + note.title);
+          console.log("editor.write heading " + note.id);
 
           updatedHeadingCount += 1;
           const start = new Position(
-            heading.range.start.line,
-            heading.range.start.column
+            heading.range.start.line - 1,
```

Comment:
out of curiosity, what has changed that caused this?

Reply:
The AST node positions (line and column) are `1-indexed` while VS Code APIs are `0-indexed`.

To maintain this consistency I am initializing `note.start` by 1-indexed values - 

https://github.com/foambubble/foam/blob/59eb48abd7ec086dbed1a600c3f76fde0fe4d5e6/packages/foam-core/src/markdown-provider.ts#L44

Reply:
we could add a small utility function called `astPositionToVsCodePosition` which does this simple operation. 
it would both explicitly call out the intent of the operation, centralize the logic, and clean up the code on the client (caller) side.
Basically the code above would become:
```
const start = astPostionToVsCodePosition(heading.range.start)
```
(if necessary you can also create the reverse `vsCodePositionToAstPosition`, for the same reasons)

wdyt?